pkg/inngest/inngest/_internal/server_lib/execution_request.py

```diff
@@ -18,6 +18,7 @@ class ServerRequest(types.BaseModel):
 class ServerRequestCtx(types.BaseModel):
     attempt: int
     disable_immediate_execution: bool
+    max_attempts: typing.Optional[int] = None
```

Comment:
```suggestion
    max_attempts: int | None = None
```

I still need to make this change all of the SDK 😄. We can use the pipe operator for unions since we dropped 3.9 support ([PEP 604](https://peps.python.org/pep-0604))